cypress/integration/homepage.spec.js

```diff
@@ -0,0 +1,28 @@
+/// <reference types="cypress" />
+
+
+context('Homepage', () => {
```

Comment:
While I appreciate the lack of specificity so we can freely make changes, we should probably add checks for most of the links on the homepage.